### PR TITLE
Turns out `@patch('pkgdb2.lib.utils.get_packagers')` does not work

### PR DESCRIPTION
**Pull request by pypingou**
We can patch the whole module and set a return_value for one method in it
but patching the method directly doesn't work.

So we need to create some logs manually to test the timeline endpoint

### CODE REVIEW

File: tests/test_flask_ui_packages.py [hunk 1]
```diff
@@ -496,12 +496,13 @@ def test_package_retire(self, login_func, mock_func):
...
                 in output.data)
 
-    @patch('pkgdb2.lib.utils.get_packagers')
+    @patch('pkgdb2.lib.utils')
     @patch('pkgdb2.packager_login_required')
     def test_package_take(self, login_func, mock_func):
         """ Test the package_take function. """
         login_func.return_value = None
-        mock_func.return_value = ['pingou', 'toshio']
+        mock_func.get_packagers.return_value = ['pingou', 'toshio']
```
Comment (ralphbean) [inline tests/test_flask_ui_packages.py]: FWIW, ``mock_func`` should probably be renamed to something like ``utils_module`` or ``utils_mod``.

Comment (pypingou) [inline tests/test_flask_ui_packages.py]: Yes it should, in the whole file :-)

**Comment by ralphbean** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks @ralphbean for the review

**Pull request merged**